crates/spark/src/operator/rpc/spark_rpc_client.rs

```diff
@@ -667,7 +667,14 @@ impl SparkRpcClient {
         let valid_session = match current_session {
             Ok(session) => self.auth.get_authenticated_session(Some(session)).await,
             Err(e) => {
-                error!("Failed to get operator session from session manager: {}", e);
+                match e {
```

Comment:
could you add the same to the graphql client?
https://github.com/breez/spark-sdk/blob/62eb94e7f5e85f1420ff91784ff621a3c8aba99c/crates/spark/src/ssp/graphql/client.rs#L500-L503